.github/workflows/bump_version.yml

```diff
@@ -24,16 +24,14 @@ jobs:
           perl -i -pe 's/^(templateScriptVersion=(.*))$/"templateScriptVersion=${{ github.event.inputs.newVersion }}"/e' version.properties
 
       - name: Bump version XML
-        uses: chkfung/android-version-actions@v1.2.1
-        with:
-          gradlePath: sample-xml/app/build.gradle.kts
-          versionName: ${{ github.event.inputs.newVersion }}
+        id: set_version_xml
```

Comment:
@doannimble Just curious, what value does `id` bring? 🙏

Reply:
@Tuubz nothing, I've removed them. Thanks for your question 🙏